README.md

```diff
@@ -210,6 +210,35 @@ console.log(
 );
 ```
 
+## File Size Parsing
+The `FileSizeFormat` object can be used to format a file size appropriately for the user's locale.
+
+Syntax:
+
+```javascript
+var formatFS = new d2lIntl.FileSizeFormat(locale[, options]);
+```
+Options:
+
+- **locale**: see [overriding locale data](#Overriding Locale Data)
+- **decimalPlaces**: Change the default maximum amount of decimal places to include in the result (Default: 2)
```

Comment:
I'm going to update the decimalPlaces readme text to maximumFractionDigits in the next PR